internal/services/orbital/contact_resource_test.go

```diff
@@ -55,7 +55,7 @@ func (r ContactResource) basic(data acceptance.TestData) string {
 resource "azurerm_orbital_contact" "test" {
   name                   = "testcontact-%[2]d"
   resource_group_name    = azurerm_resource_group.test.name
-  spacecraft_id          = azurerm_orbital_spacecraft.test.id
+  spacecraft_id          = "/subscriptions/1a6092a6-137e-4025-9a7c-ef77f76f2c02/resourceGroups/spaceymcspaceface/providers/Microsoft.Orbital/spacecrafts/spaceymcspaceface"
```

Comment:
could we have this id as an env var that gets passed in instead? For example, like these tests have: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/bot/bot_channel_facebook_resource_test.go